webapp-django/crashstats/documentation/templates/documentation/supersearch/home.html

```diff
@@ -0,0 +1,389 @@
+
+            <pre><code>{
+    "facet_name": [
+        {
+            "term": "foo",
+            "count": 42,
+            ["facets": {
+                "other_facet_name": []
+            }]
+        }
+    ]
+}</code></pre>
+
+            <p>Aggregations can theoratically be infinitely nested, but are currently limited by the available arguments to 3 levels. </p>
+
+            <p>There are 3 ways of getting aggregations. The first and simplest one is to use <code>_facets</code>. The second is to use <code>_aggs.field_name</code> and its derivative, like <code>_aggs.field1.field2</code>. And the third one is to use special aggregations, like <code>_histogram.field_name</code> or <code>_cardinality.field_name</code>. </p>
+
+            <h2>Basic aggregations</h2>
+
+            <p>Using the <code>_facets</code> parameter allows you to run simple aggregations. You can pass it a field name (see the <a href="{{ url('documentation:supersearch_api') }}#section-list-of-fields">List of fields</a>), and it will count the terms of that field in all the documents of the dataset. It will then return them sorted by descending count. </p>
+
+            <div class="example">
+                <p>Example</p>
+                <pre><code><span class="http-verb">GET</span> <span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span><span class="query-string">?_facets=signature</span></code></pre>
+            </div>
+
+            <p>This will return the list of signatures that appear the most in all crash reports for the last week, sorted by descending count. </p>
+
+            <p>The number of results in an aggregation is 50 by default. If you want to change it, you can use the <code>_facets_size</code> parameter. Note that there is no way to paginate over aggregation results, and that the bigger the <code>_facets_size</code> the longer the request will take. </p>
+
+            <h2>Nested aggregations</h2>
+
+            <p>Let's say you want to get the count of platforms per product. To do that, you would use a nested aggregation: aggregate on products, and then for each product aggregate on platforms. Here's what that translates to in Super Search: </p>
+
+            <div class="example">
+                <p>Example</p>
+                <pre><code><span class="http-verb">GET</span> <span class="url-domain">{{ full_url(request, 'api:model_wrapper', model_name='SuperSearch') }}</span><span class="query-string">?_aggs.product=platform</span></code></pre>
+            </div>
+
+            <h2>Special aggregations</h2>
+
+            <div class="examples">
+            <h1 id="examples">Examples</h1>
+
+            <h2>List of signatures</h2>
+
+            <h3>Goal</h3>
+            <p>Get the top 1000 signatures for Firefox 2.0. </p>
+
```

Comment:
Perhaps I'm missing something but it would be awesome if this was a real example that I can copy and paste into a `curl` run. You could get the latest version of Firefox from the context (in the django view) and you can get the word "Firefox" from `settings.DEFAULT_PRODUCT` instead. 

And 1,000 is not a great example because if you run this on `curl` you'll get an overwhelming result output.